### DISCUSSION TITLE
Wrong in Wiki

### DISCUSSION THREAD

**Discussion by PhPersson** [homeassistant-apps/app-cloudflared#758] [Ideas]
The wiki is of configurinng [remote tunnels](https://github.com/brenner-tobias/addon-cloudflared/wiki/How-tos#how-to-configure-remote-tunnels) is showing an old image of cloudflares zero trust. The tunnels are now under _Network - Tunnels_. 

Here's a new updated image
![1](https://github.com/user-attachments/assets/d693f9f3-6f4a-47cf-adae-5d59da6bd20b)

**Comment by elcajon** [comment]
#839 will update the image. Sorry for the late response and thanks a lot!
